mmpose/models/backbones/tcn.py

```diff
@@ -74,24 +76,27 @@ def __init__(self,
                 kernel_size=kernel_size,
                 stride=self.stride,
                 dilation=self.dilation,
-                bias=False,
+                bias='auto',
                 conv_cfg=conv_cfg,
-                norm_cfg=norm_cfg), nn.Dropout(self.dropout))
+                norm_cfg=norm_cfg))
         self.conv2 = nn.Sequential(
             ConvModule(
                 mid_channels,
                 out_channels,
                 kernel_size=1,
-                bias=False,
+                bias='auto',
                 conv_cfg=conv_cfg,
-                norm_cfg=norm_cfg), nn.Dropout(self.dropout))
+                norm_cfg=norm_cfg))
 
         if residual and in_channels != out_channels:
             self.short_cut = build_conv_layer(conv_cfg, in_channels,
                                               out_channels, 1)
         else:
             self.short_cut = None
 
+        if dropout > 0:
+            self.dropout_layer = nn.Dropout(dropout)
```

Comment:
self.dropout_layer -> self.dropout

Reply:
Set self.dropout_layer to None if dropout ratio is zero, and use `if self.dropout is None` to determine if there are droput layers